### ISSUE TITLE
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, in Admin screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#863]
**Describe the bug**
The additional content, which is being displayed on mouse hover, is disappearing while moving focus on it, in Admin screen.

**To Reproduce**

1. Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Mouse hover on the Plus icon, under Workflow tab. It will show "Add New Workflow" label.
3. Now try move the pointer over the "Add New Workflow, the label disappears.
4. User should be able to move pointer over the additional content (tool tip, label etc.).
 

**Expected behavior**
User should be able to move pointer over the additional content (tool tip, label etc.), and this additional content should not disappear, while moving focus on it.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
This is not a bug as there is no additional content on Hover. In application on Hover, only title is displayed. So closing this.
